### DISCUSSION TITLE
Access to azuracast database mariadb ?

### DISCUSSION THREAD

**Discussion by DJMic** [AzuraCast/AzuraCast#6865] [Q&A]
Hi,
I've installed the latest stable version 0.19.4 (Docker)
How can I access to azuracast's database ?
With older versions, I used this command : "./docker.sh db" but it seems no longer working.
I've this error 

![image](https://github.com/AzuraCast/AzuraCast/assets/156111736/7880ff13-8806-43f8-a67e-940e936af60e)

What's the right command to execute ?

**Comment by gAlleb** [comment]
Hi @DJMic.

```
cd /var/azuracast
bash docker.sh bash
mariadb -u azuracast -p
```
Well, youu can find password in`azuracast.env` file.

**Reply by DJMic** [reply]
Thanks for your answer, but it doesn't work.
I have this error
![image](https://github.com/AzuraCast/AzuraCast/assets/156111736/41cf4366-4fa7-4b5d-b0d0-8316363dd798)
:(

**Reply by gAlleb** [reply]
Well, it's a bummer. Any chance you could reinstall Azuracast and restore? Probably your mysqld isn't running.

You could also try and restart it through "Administration - Services - mariadb - restart" on web-interface.

There is a chance that there are some permission errors cause /run/mysqld/mysqld.sock seems right to me and exactly what is declared in /etc/mysql/my.cnf.

**Reply by DJMic** [reply]
I have restarted mariadb but I think there's a problem with permissions to access my.cnf : 
![image](https://github.com/AzuraCast/AzuraCast/assets/156111736/24e0ed00-1fe9-4875-9f3d-327da67368a3)

I tried to change permissions from 777 to 755 for my.cnf, but "Operation not permitted"
I have also run _./docker.sh update_ but always the same issue...

**Reply by saintcore** [reply]
Up first: Please remove any sensitive information (such as the mariadb password) when posting in public places such as this discussion :)

Now to your error: The warning related to the permissions isn't the problem imho and I would have to look into it to tell you what's going on. But anyway back to your connection error: The syntax of our cli-statement is wrong. The -p for "password" is interactive. That means pleas only type "mariadb -u azuracast -p" and then press enter. Now you will be promted to enter your password. After that you should be connected to the mysql-cli. You can then go on and for example list al databases ('show databases;'). 

When entering "mariadb -u azuracast -p <password" the cli assumes that the string behind -p is the database-name. Therefore you have a connection error (as stated in the message --> "access denied... to database <databasename>" (databasename = the password you entered). Hope this helps :)

For more information see https://mariadb.com/kb/en/mariadb-command-line-client/ . For example you could also connect with the following command (without an interactive password prompt):

`mariadb --user=user_name --password=your_password db_name`

**Comment by BusterNeece** [comment]
@DJMic This is a bug and should be an issue. Track the issue at #6869.
